ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request or enhancement suggestion
+about: Suggest an idea for this project
+title: ''
+labels: enhancement, triage
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? Please describe.
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when ... -->
+
+## Describe the solution you'd like
+<!-- A clear and concise description of what you want to happen. -->
+
+## Describe alternatives you've considered
+<!-- A clear and concise description of any alternative solutions or features you've considered. -->
+
+## Additional context
+<!-- Add any other context or screenshots about the feature request here. -->
+
+## Don't forget to add labels!
+- component label (`neofs-adm`, `neofs-storage`, ...)
```

Comment:
Add an example of component label from some other project, e.g. from NeoGo?

Reply:
Yeah, this should be generic enough (`see project-specific labels`, for example).